src/FplBot.EventHandlers.Discord/GameweekFinishedHandler.cs

```diff
@@ -63,8 +63,12 @@ public async Task Handle(PublishGameweekFinishedToGuild message, IMessageHandler
                     new ("ℹ️ Gameweek finished!",intro),
                     new ("ℹ️ Standings", standings),
                     new ("ℹ️ Top 3", topThree),
-                    new ("ℹ️ Lantern beige", worst)
                 });
+
+                if (worst is not null)
+                {
+                    messages.Add(new("ℹ️ Lantern beige", worst));
```

Comment:
Is "Lantern beige" some slang I'm not cool enough to understand?

Reply:
https://en.wikipedia.org/wiki/Lanterne_rouge